src/from-gpx/tracepoints-converter.ts

```diff
@@ -5,8 +5,21 @@ import * as fastXmlParser from "fast-xml-parser";
 import { RoadSnapTracePoint } from "@aws-sdk/client-geo-routes";
 
 /**
- * It converts a GPX string to an array of RoadSnapTracePoint, so the result can be used to assemble the request to
- * SnapToRoads API.
+ * It converts a GPX string with trkType to an array of RoadSnapTracePoint, so the result can be used to assemble the
+ * request to SnapToRoads API.
+ *
+ * Each trace point in the GPX can include the following information:
```

Comment:
```suggestion
 * Each track point in the GPX can include the following information:
```

(to align with GPX terminology)